py3status/modules/whatismyip.py

```diff
@@ -17,9 +17,10 @@
     hide_when_offline: hide the module output when offline (default False)
     icon_off: what to display when offline (default '■')
     icon_on: what to display when online (default '●')
+    ignore_warnings: ignore negative_cache_timeout warnings (default False)
```

Comment:
I would not add this option, even this with this PR you get all py3status users to use the same UA (more on that later)

Reply:
please remove it then ;)